packages/core/src/components/Popover/index.tsx

```diff
@@ -0,0 +1,308 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React from 'react';
+import { v4 as uuid } from 'uuid';
+import Text from '../Text';
+import withStyles, { WithStylesProps } from '../../composers/withStyles';
+import { styleSheetPopover } from './styles';
+import Portal from '../Portal';
+const EMPTY_TARGET_RECT: ClientRect = {
+  bottom: 0,
+  height: 0,
+  left: 0,
+  right: 0,
+  top: 0,
+  width: 0,
+};
+export type PopoverProps = {
+  /** Accessibility label. If not specified, all popover content is duplicated, rendered in an off-screen element with a separate layer. */
+  accessibilityLabel?: string;
+  /** Inline content to hover. */
+  children: NonNullable<React.ReactNode>;
+  /** What to show in the popover. */
+  content: NonNullable<React.ReactNode>;
+  /** True to disable popover but still show children. */
+  disabled?: boolean;
+  /** Manually override calculated horizontal align */
+  horizontalAlign?: 'center' | 'left' | 'right';
+  /** True to use a light background with dark text. */
+  inverted?: boolean;
+  /** Number of seconds the cursor must hover over the trigger to trigger the popoover */
+  mouseEnterDelay?: number;
+  /** Number of seconds popover will persist if you do not move your cursor into the popover. */
+  mouseLeaveDelay?: number;
+  /** Callback fired when the popover is shown. */
+  onShow?: () => void;
+  /** True to prevent dismissmal on mouse down. */
+  remainOnMouseDown?: boolean;
+  /** True to toggle popover display on click.  */
+  toggleOnClick?: boolean;
+  /** True to add a dotted bottom border. */
+  underlined?: boolean;
+  /** Manually override calculated vertical align */
+  verticalAlign?: 'above' | 'below';
+  /** Width of the popover in units. */
+  width?: number;
+};
+export type PopoverState = {
+  labelID: string;
+  open: boolean;
+  popoverHeight: number;
+  targetRect: ClientRect;
+  targetRectReady: boolean;
+};
+export type PositionStruct = {
+  above: boolean;
+  align: 'left' | 'center' | 'right';
+};
+export type StyleStruct = {
+  center: string | number;
+  right: string | number;
+};
+/** A simple popover component */
+export class Popover extends React.Component<PopoverProps & WithStylesProps, PopoverState> {
+  static inverted: boolean = false;
```

Comment:
curious why this needed to be added as static? inverted is falsey (false or undefined) by default.

Reply:
actually, looking over this and Tooltip, they share _a _lot__ of logic. did you explore using Tooltip under the hood and adding private props to Tooltip instead to make this work?

https://github.com/airbnb/lunar/blob/master/packages/core/src/components/Tooltip/index.tsx

Reply:
Curious, why not add these props to Tooltip instead? Why a separate component? Visually they look the same, just added functionality.

Reply:
The popup portion of the Tooltip component also has a giant <Overlay> wrapping the <span> that prevents copy-paste and clicking inside the popup, whereas this Popover does not. Otherwise, I agree -- this component is basically Tooltip with a few new props, a timer, and no overlay, and it should be a lot simpler. 

Which direction would you recommend with this context? Importing / extending Tooltip props? Modifying Tooltip? Or continuing with the separate Popover component?